### PR TITLE
Std enkf debug

### PR DESCRIPTION
**Pull request by joakim-hove**
Custom analysis module to facilitate arbitrary debugging.

### CODE REVIEW

**Comment by jepebe** [comment]
Nice!!

**Comment by joakim-hove** [comment]
This has been tested and used internally; and seems to work as anticipated.

**Comment by joakim-hove** [comment]
Merging any day!

File: python/tests/ert/analysis/test_std_enkf_debug.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
+#  Copyright (C) 2016  Statoil ASA, Norway.
+#
+#  The file 'test_analysis_module.py' is part of ERT - Ensemble based Reservoir Tool.
```
Comment (pgdr) [inline python/tests/ert/analysis/test_std_enkf_debug.py]: "This file is part of ERT"

File: libanalysis/modules/std_enkf_debug.c [hunk 2]
```diff
@@ -0,0 +1,247 @@
+/*
+   Copyright (C) 2011  Statoil ASA, Norway.
+
+   The file 'std_enkf.c' is part of ERT - Ensemble based Reservoir Tool.
```
Comment (pgdr) [inline libanalysis/modules/std_enkf_debug.c]: "this file*" and 2011→2016

**Pull request merged**